src/config.js

```diff
@@ -11,6 +11,9 @@ const DISCORD_SERVER = {
   scheduledDiscordEventsEnabled:
     process.env.SCHEDULED_DISCORD_EVENTS_ENABLED === 'true',
 };
+const LISTEN_NEW_EVENTS_REPORT_CHANNEL = {
+  announcements: process.env.ANNOUNCEMENTS_CHANNEL_ID,
+};
```

Comment:
what if we need to add more things on this config after?, it could look weird as the constant name says report channel. Maybe changing the name this way would look better:
```suggestion
const LISTEN_NEW_EVENTS = {
  report_channel: process.env.ANNOUNCEMENTS_CHANNEL_ID,
};
```